Libs/Core/ctkAbstractWorker.h

```diff
@@ -73,6 +74,7 @@ public slots:
 protected:
   QSharedPointer<ctkAbstractJob> Job;
   QSharedPointer<ctkJobScheduler> Scheduler;
+  QMutex mMutex;
```

Comment:
Could it have a more specific name? E.g., if it prevents concurrent access to "Job" object then it would be "JobMutex".